lib/src/model/sdk_publish_model.dart

```diff
@@ -175,54 +175,44 @@ class SDKPublishLayout extends PrivacyBase {
   /// The layout's unique id.
   final String id;
 
-  /// The layout's canvas id.
-  final String canvasId;
-
   /// The layout's page id.
   final String pageId;
 
   /// The layout's project id.
   final String projectId;
 
   /// A list of the nodes that makes up this layout.
-  @NodesMapConverter()
-  final Map<String, BaseNode> nodes;
+  /// CanvasID -> <NodeID -> Node>
+  @CanvasesMapConverter()
```

Comment:
I don't believe this is necessary.... Is it?

Reply:
Yes it is. We did test this on a meeting together and it was failing.